iModelCore/ECDb/ECDb/ECDbSqlFunctions.cpp

```diff
@@ -22,6 +22,7 @@ void PropExistsFunc::_ComputeScalar(Context& ctx, int nArgs, DbValue* args) {
         return;
     }
 
+    if (m_propMap.Empty()) m_propMap.Build(m_ecdb, true);

```

Comment:
Can two threads call `_ComputeScalar` on the same `PropExistsFunc` object simultaneously?

Reply:
There is a case where you can copy a SqlFunc to two different connections and in that case `_ComputeScalar` will be called by two different threads.  But we do not do that anywhere in ECDb. ECPresentation does that by registering the same function across multiple connections.